internal/app/connectconformance/testsuites/data/tls-client-certs.yaml

```diff
@@ -4,7 +4,7 @@ reliesOnTlsClientCerts: true
 # This just does the basics with a client-cert, instead of running every test case with them.
 testCases:
   - request:
-      testName: client stream success
+      testName: client-stream/success
```

Comment:
I don't think the "success" suffix is useful since these really all would be basic success tests. Also, maybe a TODO to add unary and other stream type test cases here.